src/_tests/fixtures/43960/derived.json

```diff
@@ -0,0 +1,45 @@
+{
+  "type": "info",
+  "now": "2020-04-29T22:09:53.999Z",
+  "pr_number": 43960,
+  "updatedAt": "2020-04-29T20:48:53.000Z",
+  "author": "aaltepet",
+  "owners": [],
+  "dangerLevel": "ScopedAndTested",
+  "headCommitAbbrOid": "129f84e",
+  "headCommitOid": "129f84e4492a76e7cd5af9d946c4583a60e6eb88",
+  "mergeIsRequested": false,
+  "stalenessInDays": 13,
+  "lastCommitDate": "2020-04-16T17:21:54.000Z",
+  "reopenedDate": "2020-04-29T20:48:52.000Z",
+  "lastAuthorCommentDate": "2020-04-29T20:46:05.000Z",
```

Comment:
You can see the last push was a while ago, but the author just asked me a question today, so this now counts as being fresh. Also, I reopened the PR after typescript-bot closed it, and I’m also counting that as fresh activity.